context/app/api/vitessce_confs/assay_confs.py

```diff
@@ -102,7 +105,7 @@ class CytokitSPRMConf(ViewConf):
     @return_empty_json_if_error
     def build_vitessce_conf(self):
         file_paths_found = [file["rel_path"] for file in self._entity["files"]]
-        found_tiles = get_matches(file_paths_found, TILE_REGEX)
+        found_tiles = get_matches(file_paths_found, f"{TILE_REGEX}|{STITCHED_REGEX}")
```

Comment:
```suggestion
        found_tiles = get_matches(file_paths_found, TILE_REGEX) or get_matches(file_paths_found, STITCHED_REGEX) 
```
My guess is that we don't expect to see match both `TILE_REGEX` and `STITCHED_REGEX`... but if we did, would it be better to just return one or the other? On the principle that we should try to be strict about what we return, to make the job easier downstream.